include/aspect/heating_model/interface.h

```diff
@@ -92,7 +130,7 @@ namespace aspect
         void
         evaluate (const typename aspect::MaterialModel::Interface<dim>::MaterialModelInputs &material_model_inputs,
                   const typename aspect::MaterialModel::Interface<dim>::MaterialModelOutputs &material_model_outputs,
-                  std::vector<double> &heating_outputs) const;
+                  HeatingModel::HeatingModelOutputs &heating_model_outputs) const;
```

Comment:
The documentation of this function no longer fits its arguments. 

I think the documentation should give guidance whether the two parts of the output array are supposed to be zeroed out when you enter this function. The implementations of this function currently only set one of the two output arrays. Should they have to zero out the other field, or can they assume that it is already zero?